### DISCUSSION TITLE
Reminder string on OLED

### DISCUSSION THREAD

**Discussion by gitrimius** [shanteacontrols/OpenDeck#47] [Ideas]
Hi,
I thought it would be nice to have some kind of reminder on the OLED display when one turns a potentiometer or presses a button etc... as an extension of the current value display. The reminder could come in the form of a simple string that represent the _actual_ function. For example: "Rvrb>Amount:04" or "Rvrb>Time:21". In the WebUI that would mean that one could fill in a text string next to the pot or button table line that is activated when there is action on this control.
Just a suggestion or idea...

**Comment by paradajz** [comment]
Sorry, this kind of feature isn't planned.

**Comment by gitrimius** [comment]
ok! just a suggestion!
